### PR TITLE
Add PAFS documentation

### PR DESCRIPTION
**Pull request by kennyevil**
Adds documentation for the PAFS service

### CODE REVIEW

File: services/pafs/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# Repos
+
+The service uses the following repos
+
+(Pafs-user)[https://github.com/DEFRA/pafs-user]
+(Pafs-admin)[https://github.com/DEFRA/pafs-admin]
+(Pafs_core)[https://github.com/DEFRA/pafs_core]
+
+Pafs-user is the front office application for users to submit thheir project proposals
```
Comment (timstone) [inline services/pafs/README.md]: Typo

File: services/pafs/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Pafs-admin is the back office application to administer user accounts
+Pafs_core is the common set of code used by both of the applications. Most code changes will be made here.
+
+# Updating the repos
+
+`pafs_core` has the `main` branch which is for work ready for release to production and the `develop` branch for work in progress.
+`pafs-admin` and `pafs-user` have the ```main``` branch which uses the `main` branch from `pafs_core` and the `development` branch which uses the `develop` branch from `pafs_core`.
```
Comment (timstone) [inline services/pafs/README.md]: having a development branch and develop branch is confusing IMHO

File: services/pafs/adding_new_areas.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+> name, parent area, type, sub type
+
+Areas in PAFS can be one of four types:
+
+1. RMA
+2. PSO
+3. EA Area
+4. Country
+
+RMAs will have a subtype of either LA or WC.
```
Comment (timstone) [inline services/pafs/adding_new_areas.md]: Would be good to explain what LA and WC are

File: services/pafs/adding_new_attributes_to_projects.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Adding new attributes to the POL integration
+
+The PAFS-user application sends data via a message that sends a JSON document to POL.
```
Comment (timstone) [inline services/pafs/adding_new_attributes_to_projects.md]: Explain POL accronym

File: services/pafs/preparing_for_a_release.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+Find the PRE-PROD job to deploy the app.
+
+In the `Build` box, update `CAPISTRANO_BRANCH` to the new tag (eg `v1.1.7`). Then save your changes.
+
+## Contact Web Ops
+
+Before submitting an RFC, check with Web Ops and make sure they can deploy on that date.
+
+Post in the waste carriers channel on the Web Ops slack and say when you'd like to deploy.
```
Comment (timstone) [inline services/pafs/preparing_for_a_release.md]: waste carriers reference

File: services/pafs/preparing_for_a_release.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+### Filling in the form
+
+The easiest thing to do is to look up an earlier RFC for the Configuration Item "Waste Carriers, Dealers, Brokers" and copy from that.
+
+However, if you are filling in a new form, be sure to fill in the fields as follows:
+
+- Category - "software"
+- Environment - "production"
+- Configuration item - "Waste Carriers, Dealers, Brokers".
```
Comment (timstone) [inline services/pafs/preparing_for_a_release.md]: Waste carriers reference

**Review by timstone — CHANGES_REQUESTED** [review]

**Review by jjromeo — APPROVED** [review]
LGTM

**Pull request merged**